vm/onboarding.go

```diff
@@ -0,0 +1,90 @@
+package vm
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	tp "github.com/kubearmor/KVMService/src/types"
+	"sigs.k8s.io/yaml"
+)
+
+func postHttpRequest(eventData []byte, vmAction string, address string) (string, error) {
+
+	timeout := time.Duration(5 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
+	}
+
+	request, err := http.NewRequest("POST", address+"/"+vmAction, bytes.NewBuffer(eventData))
+	request.Header.Set("Content-type", "application/json")
+	if err != nil {
+		return "", err
+	}
+
+	resp, err := client.Do(request)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	respBody, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", err
+	}
+
+	return string(respBody), err
+}
+
+func VmList(address string) error {
```

Comment:
```suggestion
func List(address string) error {
```
Exported function is under vm, so `vm.List` is fine. `vm.VmList` seems odd